compiler/qsc/src/interpret/stateful.rs

```diff
@@ -358,14 +357,38 @@ impl Interpreter {
     ) -> Result<Value, Vec<LineError>> {
         let mut result = Value::unit();
 
+        let stmt_ids = self.compile_to_stmts(line)?;
```

Comment:
This refactoring changes behavior in a way that we may not intend. I'm going to revert most of this and keep my changes contained to the new method I added.